src/js/api.js

```diff
@@ -1,26 +1,37 @@
 import fetch from 'isomorphic-fetch'
 
-function sendFetch(suffix, options = {method: 'GET'}){
-  var url = makeUrl(parseLocation(), suffix);
+let accessToken
 
+function sendFetch(suffix, options = {method: 'GET'}){
+  var location = options.noParse ? {} : parseLocation();
+  var url = makeUrl(location, suffix);
   if(typeof options.body === 'object') options.body = JSON.stringify(options.body)
+  var headers = {}
 
+  if(accessToken) headers.Authorization = 'Bearer ' + accessToken
+  console.log('requesting with access token: ', accessToken)
   var fetchOptions = {
-    method: options.method,
+    method: options.method || 'GET',
```

Comment:
Aha! I thought the same, but if you pass in your OWN object, the default won't exist! You are setting the *object* as the default, not the key.

When I first wrote it, I thought the same as you, perhaps this means using a default argument here is worse because of the confusion... that said, who doesn't like default args